cmd/llrp/llrp.go

```diff
@@ -270,6 +270,7 @@ func sendCustom(ctx context.Context, c *llrp.Client, custom string) error {
 
 	for i, cst := range *msgs {
 		log.Infof("sending custom %d", i)
+		//nolint: gosec
```

Comment:
Can we comment the linter error we are ignoring on each of these?

Reply:
I had put that in due to the pointer &cst. the Lint threw the error bellow 

```
cmd/llrp/llrp.go:274:23: G601: Implicit memory aliasing in for loop. (gosec)
                logErr(send(ctx, c, &cst, &llrp.CustomMessage{}))
```

Reply:
This should be fixed not ignored: See why and how here: https://stackoverflow.com/questions/62446118/implicit-memory-aliasing-in-for-loop/68247837